cmd/vic-machine/create/create.go

```diff
@@ -1610,6 +1610,11 @@ func splitVnetParam(p string) (vnet string, value string, err error) {
 		return
 	}
 
+	if strings.Contains(vnet, " ") && (len(mapped) == 1 || (len(mapped) == 2 && len(mapped[1]) == 0)) {
```

Comment:
Here's the debug output when you supply a vSphere network name that contains spaces without specifying a network name after the colon:

It will continuously try to retry with runtime faults. This change is to require a network alias if the vSphere network name contains a space.

> ./bin/vic-machine-linux create --target [] --name vch100 --no-tlsverify --thumbprint=[] --bridge-network bridge --container-network "VM Network With Spaces" --debug=1
May 19 2017 15:14:42.399-05:00 INFO  ### Installing VCH ####
May 19 2017 15:14:42.400-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).processParams:529]
May 19 2017 15:14:42.400-05:00 WARN  Using administrative user for VCH operation - use --ops-user to improve security (see -x for advanced help)
May 19 2017 15:14:42.400-05:00 DEBUG client network: IP {<nil> <nil>} gateway <nil> dest: []
May 19 2017 15:14:42.400-05:00 DEBUG public network: IP {<nil> <nil>} gateway <nil> dest: []
May 19 2017 15:14:42.400-05:00 DEBUG management network: IP {<nil> <nil>} gateway <nil> dest: []
May 19 2017 15:14:42.400-05:00 DEBUG VCH DNS servers: []
May 19 2017 15:14:42.400-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).loadCertificates:1025]
May 19 2017 15:14:42.400-05:00 DEBUG Unable to locate existing server certificate in cert path
May 19 2017 15:14:42.400-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).loadCertificates:1025] [25.93µs]
May 19 2017 15:14:42.400-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).generateCertificates:1162]
May 19 2017 15:14:42.400-05:00 INFO  Generating self-signed certificate/key pair - private key in vch100/server-key.pem
May 19 2017 15:14:42.400-05:00 DEBUG [BEGIN] [github.com/vmware/vic/pkg/certificate.CreateSelfSigned:280]
May 19 2017 15:14:42.590-05:00 DEBUG [BEGIN] [github.com/vmware/vic/pkg/certificate.createCertificate:173]
May 19 2017 15:14:42.593-05:00 DEBUG [ END ] [github.com/vmware/vic/pkg/certificate.createCertificate:173] [3.662098ms]
May 19 2017 15:14:42.593-05:00 DEBUG [ END ] [github.com/vmware/vic/pkg/certificate.CreateSelfSigned:280] [193.370308ms]
May 19 2017 15:14:42.593-05:00 DEBUG [BEGIN] [github.com/vmware/vic/pkg/certificate.saveCertificate:207]
May 19 2017 15:14:42.593-05:00 DEBUG [ END ] [github.com/vmware/vic/pkg/certificate.saveCertificate:207] [124.036µs]
May 19 2017 15:14:42.593-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).generateCertificates:1162] [193.638745ms]
May 19 2017 15:14:42.593-05:00 WARN  Configuring without TLS verify - certificate-based authentication disabled
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).processVolumeStores:914]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).processVolumeStores:914] [12.836µs]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/create.(*Create).processParams:529] [193.997561ms]
May 19 2017 15:14:42.594-05:00 DEBUG --target=https://192.168.213.129
May 19 2017 15:14:42.594-05:00 DEBUG --thumbprint=05:CE:5A:66:5E:C7:70:9A:65:78:C5:39:2A:D1:E5:79:09:27:C2:4F
May 19 2017 15:14:42.594-05:00 DEBUG --name=vch100
May 19 2017 15:14:42.594-05:00 DEBUG --bridge-network=bridge
May 19 2017 15:14:42.594-05:00 DEBUG --container-network=VM Network With Spaces
May 19 2017 15:14:42.594-05:00 DEBUG --no-tlsverify=true
May 19 2017 15:14:42.594-05:00 DEBUG --debug=1
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).CheckImagesFiles:76]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).checkImageVersion:164]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).GetImageVersion:132]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).GetImageVersion:132] [24.463µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190] [7.57µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190] [7.009µs]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).checkImageVersion:164] [80.911µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).checkImageVersion:164]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).GetImageVersion:132]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).GetImageVersion:132] [24.31µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190] [7.613µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).getNoCommitHashVersion:190] [9.943µs]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).checkImageVersion:164] [103.115µs]
May 19 2017 15:14:42.594-05:00 DEBUG [ END ] [github.com/vmware/vic/cmd/vic-machine/common.(*Images).CheckImagesFiles:76] [257.687µs]
May 19 2017 15:14:42.594-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.NewValidator:82]
May 19 2017 15:14:42.691-05:00 DEBUG Session Environment Info:
May 19 2017 15:14:42.691-05:00 DEBUG vSphere resource cache populating...
May 19 2017 15:14:42.699-05:00 DEBUG Cached dc:
May 19 2017 15:14:42.710-05:00 DEBUG Cached cluster:
May 19 2017 15:14:42.717-05:00 DEBUG Cached ds:
May 19 2017 15:14:42.732-05:00 DEBUG Cached host:
May 19 2017 15:14:42.746-05:00 DEBUG Cached pool:
May 19 2017 15:14:42.746-05:00 DEBUG vSphere resource cache populated...
May 19 2017 15:14:42.746-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.NewValidator:82] [152.118055ms]
May 19 2017 15:14:42.746-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).Validate:277]
May 19 2017 15:14:42.746-05:00 INFO  Validating supplied configuration
May 19 2017 15:14:42.747-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).basics:324]
May 19 2017 15:14:42.747-05:00 DEBUG Setting scratch image size to 8000000 KB in VCHConfig
May 19 2017 15:14:42.747-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).basics:324] [54.061µs]
May 19 2017 15:14:42.747-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).target:368]
May 19 2017 15:14:42.747-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).managedbyVC:375]
May 19 2017 15:14:42.767-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).managedbyVC:375] [20.208176ms]
May 19 2017 15:14:42.767-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).target:368] [20.250673ms]
May 19 2017 15:14:42.843-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).compute:33]
May 19 2017 15:14:42.843-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).ResourcePoolHelper:60]
May 19 2017 15:14:42.843-05:00 DEBUG Using default resource pool for compute resource: "/ha-datacenter/host/localhost.localdomain/Resources"
May 19 2017 15:14:42.843-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).ResourcePoolHelper:60] [83.892µs]
May 19 2017 15:14:42.843-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).compute:33] [150.182µs]
May 19 2017 15:14:42.843-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).storage:36]
May 19 2017 15:14:42.843-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).DatastoreHelper:98]
May 19 2017 15:14:42.850-05:00 INFO  Using default datastore: datastore1
May 19 2017 15:14:42.860-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).DatastoreHelper:98] [16.961821ms]
May 19 2017 15:14:42.860-05:00 DEBUG No path element specified for image store - will use default
May 19 2017 15:14:42.860-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).storage:36] [17.081197ms]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).network:187]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).portGroupConfig:89]
May 19 2017 15:14:42.860-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).portGroupConfig:89] [17.134µs]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).checkPortGroups:113]
May 19 2017 15:14:42.860-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).checkPortGroups:113] [19.902µs]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).configureSharedPortGroups:164]
May 19 2017 15:14:42.860-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).configureSharedPortGroups:164] [14.403µs]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40]
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] VM Network
May 19 2017 15:14:42.860-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] VM Network
May 19 2017 15:14:42.876-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] [15.818769ms] VM Network
May 19 2017 15:14:42.876-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] [15.95009ms] VM Network
May 19 2017 15:14:42.876-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40] [15.993501ms]
May 19 2017 15:14:42.876-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40]
May 19 2017 15:14:42.876-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] VM Network
May 19 2017 15:14:42.876-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] VM Network
May 19 2017 15:14:42.894-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] [17.373455ms] VM Network
May 19 2017 15:14:42.894-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] [17.432614ms] VM Network
May 19 2017 15:14:42.894-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40] [17.491253ms]
May 19 2017 15:14:42.894-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40]
May 19 2017 15:14:42.894-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] VM Network
May 19 2017 15:14:42.894-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] VM Network
May 19 2017 15:14:42.909-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] [15.540742ms] VM Network
May 19 2017 15:14:42.910-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).networkHelper:480] [15.627335ms] VM Network
May 19 2017 15:14:42.910-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getEndpoint:40] [15.658136ms]
May 19 2017 15:14:42.910-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).dpgHelper:519] bridge
May 19 2017 15:14:42.910-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] bridge
May 19 2017 15:14:42.923-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] [12.930763ms] bridge
May 19 2017 15:14:42.923-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).dpgHelper:519] [13.013197ms] bridge
May 19 2017 15:14:42.923-05:00 DEBUG Network configuration:
May 19 2017 15:14:42.923-05:00 DEBUG 	Network: public NetworkEndpoint: &{{  public } false <nil> {<nil> <nil>} {{ Network:HaNetwork-VM Network public }  [] {<nil> <nil>} true [] [] [] {{<nil> <nil>} []}} []}
May 19 2017 15:14:42.923-05:00 DEBUG 	Network: client NetworkEndpoint: &{{  client } false <nil> {<nil> <nil>} {{ Network:HaNetwork-VM Network client }  [] {<nil> <nil>} false [] [] [] {{<nil> <nil>} []}} []}
May 19 2017 15:14:42.923-05:00 DEBUG 	Network: management NetworkEndpoint: &{{   } false <nil> {<nil> <nil>} {{ Network:HaNetwork-VM Network management }  [] {<nil> <nil>} false [] [] [] {{<nil> <nil>} []}} []}
May 19 2017 15:14:42.923-05:00 DEBUG 	Network: bridge NetworkEndpoint: &{{ Network:HaNetwork-bridge bridge } true <nil> {<nil> <nil>} {{ Network:HaNetwork-bridge bridge } bridge [] {<nil> <nil>} false [] [] [] {{<nil> <nil>} []}} []}
May 19 2017 15:14:42.923-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).checkVDSMembership:552] HaNetwork-bridge
May 19 2017 15:14:42.923-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).checkVDSMembership:552] [15.168µs] HaNetwork-bridge
May 19 2017 15:14:42.923-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).dpgHelper:519] VM Network With Spaces
May 19 2017 15:14:42.923-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] VM Network With Spaces
May 19 2017 15:14:42.939-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).getNetwork:462] [16.368835ms] VM Network With Spaces
May 19 2017 15:14:42.939-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).dpgHelper:519] [16.428123ms] VM Network With Spaces
May 19 2017 15:14:42.939-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).checkVDSMembership:552] HaNetwork-VM Network With Spaces
May 19 2017 15:14:42.939-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).checkVDSMembership:552] [12.278µs] HaNetwork-VM Network With Spaces
May 19 2017 15:14:42.939-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).nicNumbers:430]
May 19 2017 15:14:42.939-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).nicNumbers:430] [12.414µs]
May 19 2017 15:14:42.939-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).network:187] [79.26819ms]
May 19 2017 15:14:42.939-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).CheckFirewall:85]
May 19 2017 15:14:42.939-05:00 DEBUG Checking firewall with management network IP {<nil> <nil>}
May 19 2017 15:14:42.980-05:00 INFO  Firewall status: ENABLED on "/ha-datacenter/host/localhost.localdomain/localhost.localdomain"
May 19 2017 15:14:42.989-05:00 DEBUG filtered rules: [{{} vSPC VM serial port connected to vSPC false [{{} 0 65535 outbound dst tcp}]  true 0xc420511b40}]
May 19 2017 15:14:42.989-05:00 DEBUG filtered IPs: [] networks: [] allIP: true rule: vSPC
May 19 2017 15:14:42.989-05:00 INFO  Firewall configuration OK on hosts:
May 19 2017 15:14:42.989-05:00 INFO  	"/ha-datacenter/host/localhost.localdomain/localhost.localdomain"
May 19 2017 15:14:42.989-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).CheckFirewall:85] [49.996178ms]
May 19 2017 15:14:43.017-05:00 WARN  Evaluation license detected. VIC may not function if evaluation expires or insufficient license is later assigned.
May 19 2017 15:14:43.017-05:00 INFO  License check OK
May 19 2017 15:14:43.017-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).CheckDrs:504]
May 19 2017 15:14:43.017-05:00 INFO  DRS check SKIPPED - target is standalone host
May 19 2017 15:14:43.017-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).CheckDrs:504] [26.044µs]
May 19 2017 15:14:43.017-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).certificate:458]
May 19 2017 15:14:43.017-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).certificate:458] [397.463µs]
May 19 2017 15:14:43.017-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).certificateAuthorities:477]
May 19 2017 15:14:43.017-05:00 DEBUG Configuring without client verification due to empty certificate authorities
May 19 2017 15:14:43.017-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).certificateAuthorities:477] [21.862µs]
May 19 2017 15:14:43.017-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).registries:502]
May 19 2017 15:14:43.017-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).registries:502] [6.728µs]
May 19 2017 15:14:43.017-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).compatibility:522]
May 19 2017 15:14:43.039-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).checkDatastoresAreWriteable:577]
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).checkDatastoresAreWriteable:577] [28.91024ms]
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).compatibility:522] [50.503682ms]
May 19 2017 15:14:43.068-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).syslog:667]
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).syslog:667] [14.954µs]
May 19 2017 15:14:43.068-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).ListIssues:252]
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).ListIssues:252] [11.033µs]
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).Validate:277] [321.559313ms]
May 19 2017 15:14:43.068-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/validate.(*Validator).AddDeprecatedFields:637]
May 19 2017 15:14:43.068-05:00 DEBUG Datacenter: "ha-datacenter", Cluster: "/ha-datacenter/host/localhost.localdomain", Resource Pool: ""
May 19 2017 15:14:43.068-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/validate.(*Validator).AddDeprecatedFields:637] [70.239µs]
May 19 2017 15:14:43.068-05:00 INFO
May 19 2017 15:14:43.068-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.NewDispatcher:79]
May 19 2017 15:14:43.068-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).InitDiagnosticLogsFromConf:96]
May 19 2017 15:14:43.116-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).InitDiagnosticLogsFromConf:96] [47.577076ms]
May 19 2017 15:14:43.116-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.NewDispatcher:79] [47.691512ms]
May 19 2017 15:14:43.116-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).CreateVCH:33] vch100
May 19 2017 15:14:43.116-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).checkExistence:97]
May 19 2017 15:14:43.116-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).findResourcePool:133] vch100
May 19 2017 15:14:43.157-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).findResourcePool:133] [41.216042ms] vch100
May 19 2017 15:14:43.157-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).checkExistence:97] [41.321916ms]
May 19 2017 15:14:43.157-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createPool:61]
May 19 2017 15:14:43.157-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createResourcePool:35]
May 19 2017 15:14:43.197-05:00 INFO  Creating Resource Pool "vch100"
May 19 2017 15:14:43.207-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createResourcePool:35] [49.863619ms]
May 19 2017 15:14:43.207-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createPool:61] [49.959057ms]
May 19 2017 15:14:43.207-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createBridgeNetwork:31]
May 19 2017 15:14:43.207-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createBridgeNetwork:31] [28.834µs]
May 19 2017 15:14:43.207-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createAppliance:465]
May 19 2017 15:14:43.207-05:00 INFO  Creating appliance on target
May 19 2017 15:14:43.207-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createApplianceSpec:293]
May 19 2017 15:14:43.207-05:00 DEBUG generating new config secret key
May 19 2017 15:14:43.209-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addIDEController:269]
May 19 2017 15:14:43.209-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addIDEController:269] [61.673µs]
May 19 2017 15:14:43.209-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addParaVirtualSCSIController:281]
May 19 2017 15:14:43.209-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addParaVirtualSCSIController:281] [109.084µs]
May 19 2017 15:14:43.209-05:00 DEBUG [BEGIN] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addNetworkDevices:213]
May 19 2017 15:14:43.215-05:00 DEBUG Setting "public" to slot 192
May 19 2017 15:14:43.215-05:00 INFO  Network role "client" is sharing NIC with "public"
May 19 2017 15:14:43.215-05:00 INFO  Network role "management" is sharing NIC with "public"
May 19 2017 15:14:43.219-05:00 DEBUG Setting "bridge" to slot 224
May 19 2017 15:14:43.219-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).addNetworkDevices:213] [10.282445ms]
May 19 2017 15:14:43.219-05:00 DEBUG [ END ] [github.com/vmware/vic/lib/install/management.(*Dispatcher).createApplianceSpec:293] [11.796617ms]
May 19 2017 15:14:43.389-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:43.449-05:00 WARN  retrying task
May 19 2017 15:14:43.550-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:43.753-05:00 WARN  retrying task
May 19 2017 15:14:43.915-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:44.200-05:00 WARN  retrying task
May 19 2017 15:14:44.354-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:45.162-05:00 WARN  retrying task
May 19 2017 15:14:45.338-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:46.731-05:00 WARN  retrying task
May 19 2017 15:14:46.894-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:48.015-05:00 WARN  retrying task
May 19 2017 15:14:48.182-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:49.912-05:00 WARN  retrying task
May 19 2017 15:14:50.076-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:51.644-05:00 WARN  retrying task
May 19 2017 15:14:51.792-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
May 19 2017 15:14:52.848-05:00 WARN  retrying task
May 19 2017 15:14:52.937-05:00 DEBUG task retry on expected task fault: &types.InvalidArgument{RuntimeFault:types.RuntimeFault{MethodFault:types.MethodFault{FaultCause:(*types.LocalizedMethodFault)(nil), FaultMessage:[]types.LocalizableMessage(nil)}}, InvalidProperty:"config.extraConfig[\"guestinfo.vice./network/container_networks|VM Network With Spaces/Common/ExecutionEnvironment\"]"}
^C